integration_tests/pages/barcode/selectEnvelopeSize.ts

```diff
@@ -41,4 +43,12 @@ export default class SelectEnvelopeSizePage extends Page {
   c4Radio = (): PageElement => cy.get('#select-envelope-size-form input[name=envelopeSize][value=c4]')
 
   submitButton = (): PageElement => cy.get('#select-envelope-size-form button')
+
+  static goToPage = (): SelectEnvelopeSizePage => ChooseBarcodeOptionPage.goToPage().happyPathCoversheet()
+
+  happyPathDl = (): PrintCoversheetsPage => this.submitHavingSelectedDlEnvelopeSize()
```

Comment:
Multiple happy paths could begin to look messy, as I think you mentioned earlier